### PR TITLE
add icons to gatsby-config.js

### PR DESCRIPTION
**Pull request by frames75**
Building the *frontend*, when you do:

```sh
$ cd frontend
$ npm install
$ npm run develop
```

This last command returns next error:

```sh
> gatsby-starter-default@0.1.0 develop /home/carpanta/Documentos/Cosas/Curro/nodejs/strapi/temp-pullreq-strapi-starte/strapi-starter-gatsby-blog/frontend
> gatsby develop

success open and validate gatsby-configs - 0.064s

 ERROR #11331  PLUGIN

Invalid plugin options for "gatsby-plugin-manifest":

- "value" must contain at least one of [icon, icons]

not finished load plugins - 0.687s

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! gatsby-starter-default@0.1.0 develop: `gatsby develop`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the gatsby-starter-default@0.1.0 develop script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/carpanta/.npm/_logs/2020-12-21T16_12_06_775Z-debug.log
```

The Error is solved by adding to `frontend/gatsby-config.js` the properties **icon** and **icons**. 

It's not necessary to add the icons files.

------

Linux Mint 19.2
Strapi: v3.4.0
Node.js: v14.15.3

### CODE REVIEW

File: frontend/gatsby-config.js [hunk 1]
```diff
@@ -32,6 +32,21 @@ module.exports = {
...
+            src: `src/images/my-icon.png`,
+            sizes: `48x48`,
+            type: `image/png`,
+          },
+          {
+            src: `src/images/my-icon.png`,
+            sizes: `72x72`,
+            type: `image/png`,
+          },
+        ],
```
Comment (markkaylor) [inline frontend/gatsby-config.js]: Hi @frames75 thanks for your PR.  I think you can remove this array and we can use automatic mode https://www.gatsbyjs.com/plugins/gatsby-plugin-manifest/#automatic-mode-configuration.

You'll need to add the image in `src/images`.  Could you change the name to favicon.png or default-icon.png

For reference you can see how it is setup on: https://github.com/strapi/strapi-starter-gatsby-catalog/blob/main/frontend/gatsby-config.js

Comment (frames75) [reply frontend/gatsby-config.js]: That's perfect. 

I changed the *icon* parameter to `icon: 'src/images/favicon.png'` and added the `src/images/favicon.png`, obtained from the *strapi-starter-gatsby-catalog*.

File: frontend/gatsby-config.js [hunk 2]
```diff
@@ -32,6 +32,8 @@ module.exports = {
         background_color: "#663399",
         theme_color: "#663399",
         display: "minimal-ui",
+
```
Comment (markkaylor) [inline frontend/gatsby-config.js]: Sorry, but could you please remove this space 🙏

Comment (frames75) [reply frontend/gatsby-config.js]: Done.

Best regards.

**Pull request merged**